xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -548,6 +548,8 @@ void CDVDInputStreamBluray::ProcessEvent() {
     CLog::Log(LOGDEBUG, "CDVDInputStreamBluray - BD_EVENT_MENU %d",
         m_event.param);
     m_menu = (m_event.param != 0);
+    if (m_menu == false)
```

Comment:
```suggestion
    if (!m_menu)
```

Reply:
is this the preferred syntax? I find it harder to read/catch when skimming through code